### PR TITLE
Add NULL type to be validate

### PR DESCRIPTION
**Pull request by walangkaji**
I think it's can fix error `NULL` type from php function `gettype()` if value is `null`

### CODE REVIEW

File: src/Validator.php [hunk 1]
```diff
@@ -27,6 +27,7 @@ protected function builtInTypes()
             'double' => 'is_double',
             'boolean' => 'is_bool',
             'string' => 'is_string',
+            'NULL' => 'is_null',
```
Comment (bixuehujin) [inline src/Validator.php]: NULL should be lowercase to consistent with other types.

Comment (walangkaji) [reply src/Validator.php]: I just reference from this: [https://www.php.net/manual/en/function.gettype.php](https://www.php.net/manual/en/function.gettype.php)

Comment (bixuehujin) [reply src/Validator.php]: Yeah, but it doesn't need to relate to the return value of  gettype, as i think the lowercased `null` is better, which looks better.

Comment (walangkaji) [reply src/Validator.php]: so, just change `'null' => 'is_null',` to be resolved?

Comment (bixuehujin) [reply src/Validator.php]: Yeah

**Review by walangkaji — COMMENTED** [review]
just updated

**Pull request merged**